views.py

```diff
@@ -1,47 +1,63 @@
 from .apps import IntercloudConfig
 from django.db.models import Model
 from django.shortcuts import render
-from django.conf import settings
+#from django.conf import settings
 from django.apps.registry import apps
 from django.core.exceptions import ImproperlyConfigured
 from django.http.request import HttpRequest
 from django.http.response import JsonResponse, HttpResponseNotFound
+from SOSCombos import settings
+from django.core import serializers
+import json
+
 # Create your views here.
 
 def _get_product(request: HttpRequest, *args, **kwargs):
-    #print("intercloud line 12: get product", request)
+    
     try:
         index = int(request.GET.get("sly-position", -1))
     except ValueError:
```

Comment:
You can delete this try catch and leave only the index line for the get query that line shouldn't raise an error